<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>friend_request</class>
 <widget class="QDialog" name="friend_request">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Send friend request</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>140</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back to menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic1">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>881</width>
     <height>541</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>171</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>171</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="users">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Show users</string>
   </property>
  </widget>
  <widget class="QPushButton" name="friends">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Show my friends</string>
   </property>
  </widget>
  <zorder>label_pic1</zorder>
  <zorder>users</zorder>
  <zorder>textBrowser</zorder>
  <zorder>textBrowser_2</zorder>
  <zorder>friends</zorder>
  <zorder>pushButton</zorder>
  <zorder>lineEdit</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label</zorder>
  <zorder>textBrowser</zorder>
  <zorder>textBrowser_2</zorder>
  <zorder>users</zorder>
  <zorder>friends</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
